### DISCUSSION TITLE
Integrate isfdb.org as a cover source

### DISCUSSION THREAD

**Discussion by l3if3rik** [mateusz-bak/openreads#735] [Ideas]
**Is your feature request related to a problem? Please describe.**
Finding covers is sometimes difficult.

**Describe the solution you'd like**
Integrate https://www.isfdb.org/ as a cover source.

**Describe alternatives you've considered**
Searching and downloading covers from isfdb.org manually. Frustrating, since the site is not mobile friendly.

**Additional context**
isfdb.org has a lot of covers categorized by year and language. It's easy to find covers for your book, no matter what language.

**Comment by Sarat-1997** [comment]
@l3if3rik is the above cover source url work almost perfectly while searching and scanning all books?
